src/harness/harnessLanguageService.ts

```diff
@@ -249,14 +249,16 @@ export abstract class LanguageServiceAdapterHost {
     }
 }
 
+/** TypeScript Typings Installer global cache location for the tests */
+export const harnessTypingInstallerCacheLocation = "/home/src/typinginstaller/globalcache/data";
 /// Native adapter
 class NativeLanguageServiceHost extends LanguageServiceAdapterHost implements ts.LanguageServiceHost, LanguageServiceAdapterHost {
     isKnownTypesPackageName(name: string): boolean {
         return !!this.typesRegistry && this.typesRegistry.has(name);
     }
 
     getGlobalTypingsCacheLocation() {
-        return "/Library/Caches/typescript";
+        return harnessTypingInstallerCacheLocation;

```

Comment:
Are we missing any coverage by having the new values all lowercase?

Reply:
Good point I will make it with upper case